apps/designer/app/designer/features/workspace/canvas-tools/text-toolbar/text-toolbar.tsx

```diff
@@ -49,16 +64,57 @@ const onClickPreventDefault: MouseEventHandler<HTMLDivElement> = (event) => {
 
 type ToolbarProps = {
   css?: CSS;
-  onValueChange: (value: Value) => void;
   rootRef: React.Ref<HTMLDivElement>;
+  textToolbar: TextToolbarValue;
```

Comment:
I would just call it state:
```suggestion
  state: TextToolbarState;
```

Reply:
or alternatively spread it on the first level